src/Atmos/Model/bc_momentum.jl

```diff
@@ -148,28 +148,28 @@ function atmos_momentum_normal_boundary_flux_second_order!(
     atmos,
     fluxᵀn,
     n,
-    state⁻,
-    diff⁻,
-    hyperdiff⁻,
-    aux⁻,
-    state⁺,
-    diff⁺,
-    hyperdiff⁺,
-    aux⁺,
+    state_sfc⁻,
+    diffusive_sfc⁻,
+    hyperdiffusive_sfc⁻,
+    aux_sfc⁻,
+    state_sfc⁺,
+    diffusive_sfc⁺,
+    hyperdiffusive_sfc⁺,
+    aux_sfc⁺,
     bctype,
     t,
-    state1⁻,
-    diff1⁻,
-    aux1⁻,
+    state_int⁻,
```

Comment:
Suggest adding a doc-string / comments to clarify these arguments; e.g. `_int` refers to first interior node; other than that this seems reasonable.